src/MCPClient/lib/clientScripts/archivematicaCreateMETSReingest.py

```diff
@@ -441,9 +441,17 @@ def update_mets(sip_dir, sip_uuid):
     add_new_files(mets, sip_uuid, sip_dir)
     delete_files(mets, sip_uuid)
 
+    # Remove normative structMap
+    serialized = mets.serialize()
+    structmaps = serialized.findall(
+        'mets:structMap[@LABEL="Normative Directory Structure"]', namespaces=ns.NSMAP)
+    for structmap in structmaps:
+        structmap.getparent().remove(structmap)
+        print("Removed normative structMap")
+
```

Comment:
@mcantelon this approach looks better/simpler than modifying metsrw, but doesn't this need to be contingent on `createNormativeStructmap`? I mean, I think the `update_mets` function needs a kwarg like `keep_normative_structmap=True` which you set to `False` when `createNormativeStructmap` is `False` in archivematicaCreateMETS2.py.